deeppavlov/models/torch_bert/torch_transformers_nll_ranking.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+from pathlib import Path
+from typing import List, Optional, Dict, Tuple, Union, Any
+
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch import Tensor
+from transformers import AutoConfig, AutoModel, AutoTokenizer
+
+from deeppavlov.core.commands.utils import expand_path
+from deeppavlov.core.common.errors import ConfigError
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.torch_model import TorchModel
+
+log = getLogger(__name__)
+
+
+@register('torch_transformers_nll_ranker')
+class TorchTransformersNLLRanker(TorchModel):
+    """Class for ranking of relations using the model trained with NLL loss
+    Args:
+        model_name: name of the function which initialises and returns the model class
+        pretrained_bert: pretrained transformer checkpoint path or key title (e.g. "bert-base-uncased")
+        encoder_save_path: path to save the encoder checkpoint
+        linear_save_path: path to save linear layer checkpoint
+        bert_config_file: path to Bert configuration file, or None, if `pretrained_bert` is a string name
+        criterion: name of loss function
```

Comment:
criterion doesn't used anywhere and will be removed in other PR, so it's better remove from TorchTransformersNLLRanker